scapy/layers/tls/cert.py

```diff
@@ -706,11 +706,9 @@ def isRevoked(self, crl_list):
             if (self.authorityKeyID is not None and
                 c.authorityKeyID is not None and
                 self.authorityKeyID == c.authorityKeyID):
-                return self.serial in map(lambda x: x[0],
-                                                    c.revoked_cert_serials)
+                return self.serial in [x[0] for x in c.revoked_cert_serials]
```

Comment:
You can use generators instead of lists when testing `in`: `return self.serial in (x[0] for x in c.revoked_cert_serials)`